disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/exporter/ToDiskExporterBuilder.java

```diff
@@ -34,71 +33,55 @@ public List<T> deserialize(byte[] source) {
           return Collections.emptyList();
         }
       };
-  private File rootDir = new File(".");
-  private String folderName = "data";
-  private StorageConfiguration configuration = StorageConfiguration.getDefault();
-  private Clock clock = Clock.getDefault();
+
+  private final StorageBuilder storageBuilder = Storage.builder();
 
   private Function<Collection<T>, CompletableResultCode> exportFunction =
       x -> CompletableResultCode.ofFailure();
 
-  DiskExporterBuilder() {}
+  ToDiskExporterBuilder() {}
 
-  @CanIgnoreReturnValue
-  public DiskExporterBuilder<T> setRootDir(File rootDir) {
-    this.rootDir = rootDir;
-    return this;
-  }
+//  @CanIgnoreReturnValue
+//  public ToDiskExporterBuilder<T> setRootDir(File rootDir) {
```

Comment:
I think this shouldn't be commented out?

Reply:
The root dir is now part of the storage configuration, so it doesn't belong on this builder any longer (see the `build()` method for when it's used).

I'll remove this commented/boneyard tho.